presto-hive/src/main/java/com/facebook/presto/hive/InternalHiveSplit.java

```diff
@@ -38,10 +39,7 @@
 public class InternalHiveSplit
 {
     // Overhead of ImmutableList and ImmutableMap is not accounted because of its complexity.
-    private static final int INSTANCE_SIZE = ClassLayout.parseClass(InternalHiveSplit.class).instanceSize() +
-            ClassLayout.parseClass(String.class).instanceSize() +
-            ClassLayout.parseClass(String.class).instanceSize() +
-            ClassLayout.parseClass(OptionalInt.class).instanceSize();
+    private static final int INSTANCE_SIZE = ClassLayout.parseClass(InternalHiveSplit.class).instanceSize();
```

Comment:
curious: is `relativeUri` intentionally not tracked?  (since it will be shared across all splits? )

Reply:
it's tracked in getEstimatedSizeInBytes()